stryker/guides/stryker-diff-runner.md

```diff
@@ -0,0 +1,36 @@
+# stryker-diff-runner
```

Comment:
This might feel like nitpicking, but the name suggests that the stryker-diff-runner is a _Stryker test runner plugin_, while in fact it isn't a plugin at all.

How about: `stryker-diff-filter`?

Reply:
Would you mind adding a link to your git repo, so people know where to find you?

Reply:
> This might feel like nitpicking, but the name suggests that the stryker-diff-runner is a _Stryker test runner plugin_, while in fact it isn't a plugin at all.
> 
> How about: `stryker-diff-filter`?

Yeah, I did struggle with the name because before 1.X there were some warnings in the console about that. But now it is not considered as a plugin anymore if I remember well.

I choose that name because it does Run Stryker but only on the SCM diff.

If it is still a problem about plugin or not confusion I can easily rename it, no problems.